sqldelight-core-integration-tests/build.gradle

```diff
@@ -1,14 +1,14 @@
-apply plugin: 'kotlin'
+apply plugin: 'org.jetbrains.kotlin.jvm'
 
 targetCompatibility = JavaVersion.VERSION_1_7
 sourceCompatibility = JavaVersion.VERSION_1_7
 
 dependencies {
-  testImplementation project(':runtime:sqldelight-runtime:jdk')
+  testImplementation project(':runtime:sqldelight-runtime')
```

Comment:
im surprised that this works without any gradle changes. Non multiplatform modules can depend on a multiplatform modules

Reply:
Yep. I believe it works similarly to the Android plugin and variants where it negotiates what it can accept with what is provided. Since there's only one JDK thingy it matches that. Not sure how you disambiguate if there's multiple.

Reply:
we kind of do have two right, with the test target?

or is that what the `targets` part at the bottom of the `runtime/build.gradle` is configuring? that those are the only targets provided

Reply:
Possibly. I don't know enough about it yet